seatreg.php

```diff
@@ -86,4 +88,33 @@ function seatreg_plugin_deactivate() {
 }
 
 //public API
-require( 'php/public_api.php' );
\ No newline at end of file
+require( 'php/public_api.php' );
+
+//Capabilities
+function seatreg_capabilities_add() {
+	$role = get_role('administrator');
+	if ( !$role->has_cap('seatreg_manage_events') ) {
+		$role->add_cap('seatreg_manage_events');
+	}
+	if ( !$role->has_cap('seatreg_manage_bookings') ) {
+		$role->add_cap('seatreg_manage_bookings');
+	}
+}
+function seatreg_capabilities_remove() {
+	$role = get_role('administrator');
+	if ( $role->has_cap('seatreg_manage_events') ) {
+		$role->remove_cap('seatreg_manage_events');
+	}
+	if ( $role->has_cap('seatreg_manage_bookings') ) {
+		$role->remove_cap('seatreg_manage_bookings');
+	}
+}
+function seatreg_update_routine( $seatreg_db_stored_version ) {
+	if ( version_compare($seatreg_db_stored_version, '1.28', '>') ) {
+		seatreg_capabilities_add();
+	}
+}
+$seatreg_db_stored_version = get_option('seatreg_db_current_version');
+if ($seatreg_db_stored_version !== SEATREG_DB_VERSION) {
+	seatreg_update_routine($seatreg_db_stored_version);
+}
```

Comment:
This logic wont detect when plugin is upgrading. It detects when SeatReg database has changed.